lib/internal/errors.js

```diff
@@ -686,6 +686,121 @@ const fatalExceptionStackEnhancers = {
   }
 };
 
+const invalidValueTypeError = (name, expected, actual, valueType) => {
+  assert(typeof name === 'string', "'name' must be a string");
+  if (!ArrayIsArray(expected)) {
+    expected = [expected];
+  }
+
+  let msg = 'The ';
+  if (valueType !== undefined) {
+    msg += `"${name}" ${valueType} `;
```

Comment:
The only thing added to this function (along with the fourth argument), otherwise a copy of the `ERR_INVALID_ARG_TYPE` function.